casperjs 1.1.0-beta3 installs perfectly, but simply doesn't load when require()d

casperjs 1.1.0-beta3 simply doesn't load via require:

    $ npm install --save casperjs
    npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm

There are no other errors:

    $ node
    > require('casperjs')
    Error: Cannot find module 'casperjs'
        at Function.Module._resolveFilename (module.js:337:15)
        at Function.Module._load (module.js:287:25)
        at Module.require (module.js:366:17)
        at require (module.js:385:17)
        at repl:1:1
        at REPLServer.defaultEval (repl.js:251:27)
        at bound (domain.js:280:14)
        at REPLServer.runBound [as eval] (domain.js:293:12)
        at REPLServer.<anonymous> (repl.js:415:12)
        at emitOne (events.js:82:20)

    # grep version node_modules/casperjs/package.json
      "version": "1.1.0-beta3"

--- a/docs/installation.rst
+++ b/docs/installation.rst
@@ -59,8 +59,8 @@ You can install CasperJS using `npm <http://npmjs.org/>`_::
 
 .. warning::
 
-   While CasperJS is installable via npm, :ref:`it's not a NodeJS package <faq_node>`, neither it's capable to require native NodeJS modules.
-
+   While CasperJS is installable via npm, :ref:`it's not a NodeJS package <faq_node>`. **You cannot load casper by using require('casperjs') in node.**
+   
 .. index:: git
 
 Installing from git

